src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepository.java

```diff
@@ -24,17 +24,26 @@
      * @param container from where container originated
      * @param zipFileName of envelope
      * @param status of envelope
-     * @return Optional envelope
+     * @return A singleton list of envelope
```

Comment:
can you explain what the 'desired improvement' is?
If i understand correctly, this always returns either one or none items